Simulator/Test/test_conversions.py

```diff
@@ -103,4 +160,29 @@ def test_cross3():
 
 def test_norm2():
     a = np.random.randn(3)
-    np.testing.assert_allclose(np.linalg.norm(a), conv.norm2(a), atol=tol)
\ No newline at end of file
+    np.testing.assert_allclose(np.linalg.norm(a), conv.norm2(a), atol=tol)
+
+
+
+def main():
```

Comment:
`main()` actually isn't necessary, even in the context of local test debugging. To run locally, you can simply do `pytest <filename>` instead of `python <filename>`.

Reply:
Good to know. I have only used 'unittest' before, not 'pytest'. I will remove it